docs/01-overview/main-areas/telemetry/telemetry-03-traces.md

```diff
@@ -82,7 +82,25 @@ This configures the underlying OTel Collector with a pipeline for traces. The re
     http-backend      Ready     44s
     ```
 
-### 2. Switch the protocol to HTTP
+### Step 2. Enable Istio Tracing
+
+By default, the tracing feature of the Istio module is disabled to avoid increased network utilization in case of an absent of a TracePipeline.
```

Comment:
```suggestion
By default, the tracing feature of the Istio module is disabled to avoid increased network utilization in the case of the TracePipeline absence.
```